ctapipe/tools/apply_models.py

```diff
@@ -201,7 +201,7 @@ def _apply(self, reconstructor, tel_tables, start, stop):
                 write_table(
                     table[output_columns],
                     self.output_path,
-                    f"/dl2/event/telescope/{prop}/{prefix}/tel_{tel_id:03d}",
+                    f"/dl2/event/telescope/{prop.value}/{prefix}/tel_{tel_id:03d}",
```

Comment:
How is this related to the python 3.11 support?

Reply:
Checking the chabgelog, there is actually  now
https://docs.python.org/3/library/enum.html#enum.StrEnum

Which might be a better solution, will change